base/show.jl

```diff
@@ -469,6 +469,14 @@ end
 isidentifier(s::Symbol) = isidentifier(string(s))
 
 isoperator(s::Symbol) = ccall(:jl_is_operator, Cint, (Cstring,), s) != 0
+
+"""
+    operator_precedence(s::Symbol)
+
+Returns an integer representing the precedence of a Julia operator, relative
```

Comment:
Perhaps "Return" rather than "Returns", and likewise below? (See the second numbered recommendation a few paragraphs down in https://docs.julialang.org/en/stable/manual/documentation/.)